s2fft/precompute_transforms/construct.py

```diff
@@ -10,6 +10,10 @@
 from s2fft import recursions
 from warnings import warn
 
+# Maximum spin number at which Price-McEwen recursion is sufficiently accurate.
+# For spins > PM_MAX_STABLE_SPIN one should default to the Risbo recursion.
+PM_MAX_STABLE_SPIN = 5
```

Comment:
We call this PM_MAX_STABLE_SPIN but test are >=, so we're throwing errors for PM_MAX_STABLE_SPIN.  Tests should probably be >, not >=?

Reply:
I agree that this isn't the most clear, we are actually already being quite conservative here (we have stability up to closer to abs(spin) ~ 8,10 or so).

In any case I've updated the checks to be greater than and incremented the global variable to 6.